### PR TITLE
Trunk 3422

### PR DESCRIPTION
**Pull request by puevigreven**
This piece of code inserts the "View Encounter Provider" and "View Visits" privileges for roles that have "View Encounter".

### CODE REVIEW

**Comment by dkayiwa** [comment]
You will also need to squash your commits into one.

**Comment by puevigreven** [comment]
@dkayiwa please look into this, i did the squashing

**Comment by dkayiwa** [comment]
@puevigreven did you run the web application to test this changeset?

**Comment by puevigreven** [comment]
yes. i Ran the web application. First i was started in maintenance mode and asked to confirm updates. After confirmation openmrs was not able to start. But then i restarted it. Then it worked properly.

**Comment by puevigreven** [comment]
@lluismf yes, that is not standard command but I found it suitable to use it here. Am i suppose to change it? Will it cause any problem?

**Comment by puevigreven** [comment]
@dkayiwa  i have not added uuid to the "View Visits" and "View Encounter Provider" in privilege table , i didn't know what to put there. please guide me to fix it. :)

File: api/src/main/resources/liquibase-update-to-latest.xml [hunk 1]
```diff
@@ -333,6 +333,46 @@
 			constraintName="key_for_location_tag_map"/>
 	</changeSet>
 	
+
+		<changeSet id="TRUNK-3422-adding-privileges" author="PralayRamteke" >
+			<preConditions  onFail="MARK_RAN">
+   				<not>
+   				<sqlCheck expectedResult="0">
+    				SELECT count(*) FROM role_privilege  WHERE privilege='View Encounters';
```
Comment (puevigreven) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @dkayiwa  is this what you asked me to do? i removed count(*) != 0 statement. :)

**Pull request closed without merging**